__tests__/app.test.js

```diff
@@ -149,6 +149,21 @@ describe("app", () => {
           expect(body.msg).toBe("Invalid vote");
         });
     });
+    test("GET 200: returns the comment count of an article by id", () => {
+      const expectedOutput = {
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: 100,
+        comment_count: 11
+      };
+      return request(app).get('/api/articles/1').expect(200).then(({body}) => {
+        expect(body.article).toMatchObject(expectedOutput)
```

Comment:
It's brilliant that this is being done in a new test 🏆 
Because to toMatchObject can allow extra properties, and your previous test already shows that it has the other properties, for this test your expected output could only have the _comment_count_ property 💡